package.json

```diff
@@ -39,47 +26,46 @@
     "jshint": "^2.9.5",
     "minimist": "^1.2.0",
     "mocha": "^3.4.2",
-    "moustache": "0.0.2",
     "npm-run-all": "^4.1.1",
     "nsp": "^2.8.1",
-    "onchange": "^3.2.1",
     "prettier": "^1.10.2",
     "selenium-webdriver": "^3.5.0",
-    "shield-studies-addon-utils": "^4.1.0"
+    "web-ext": "^2.4.0"
   },
   "engines": {
     "node": ">=8.9.0"
   },
   "homepage": "http://github.com/mozilla/shield-studies-addon-template",
   "keywords": [
     "firefox",
-    "legacy-addon",
     "mozilla",
-    "shield-study"
+    "shield-study",
+    "template"
   ],
-  "license": "MIT",
-  "main": "index.js",
+  "license": "MPL-2",
   "repository": {
     "type": "git",
     "url": "git://github.com/mozilla/shield-studies-addon-template.git"
   },
   "scripts": {
-    "build": "bash ./bin/xpi.sh",
+    "build": "web-ext build",
+    "bundle-utils": "cp node_modules/shield-studies-addon-utils/dist/StudyUtils.jsm src/privileged/shieldUtils/jsm/StudyUtils.jsm",
     "eslint": "eslint . --ext jsm --ext js --ext json",
     "eslint-fix": "npm run eslint -- --fix",
-    "firefox": "export XPI=dist/linked-addon.xpi && npm run build && node run-firefox.js",
     "format": "prettier '**/*.{css,js,json,jsm,md}' --trailing-comma=all --ignore-path=.eslintignore --write",
-    "postformat": "npm run eslint-fix",
-    "harness_test": "export XPI=dist/linked-addon.xpi && mocha test/functional_tests.js --retry 2 --reporter json",
+    "harness_test": "export ADDON_ZIP=./dist/button_icon_preference_study_shield_study_example_-2.0.0.zip && mocha test/functional_tests.js --retry 2 --reporter json",
     "lint": "npm-run-all lint:*",
-    "lint-build:addons-linter": "# actually a post build test:  bin/addonLintTest ' + require('./package.json').name",
-    "lint:addons-linter": "addons-linter addon/webextension/",
+    "lint:addons-linter": "addons-linter src/",
     "lint:eslint": "npm run eslint",
     "lint:fixpack": "fixpack",
     "lint:nsp": "nsp check",
-    "prebuild": "cp node_modules/shield-studies-addon-utils/dist/StudyUtils.jsm addon/",
+    "lint:web-ext-lint": "web-ext lint",
```

Comment:
Interesting... Now that we're using <kbd>web-ext lint</kbd>, not sure if we need **addons-linter** or the `lint:addons-linter` script above. Output seems generally the same, with the exception of `web-ext lint` giving me a bunch of excessive debug output. Both tools report the same 7 warnings:

```sh
➜  shield-studies-addon-template git:(PR55) ✗ npm run lint:web-ext-lint

...
errors          0
notices         0
warnings        7

WARNINGS:

Code                Message                       Description                                            File            Line   Column
ICON_SIZE_INVALID   The size of the icon does     Expected icon at "icon.png" to be 48 pixels wide but   manifest.json
                    not match the manifest.       was 64.
UNSUPPORTED_API     shieldUtils.telemetry is      This API has not been implemented by Firefox.          background.js   41     7
                    not supported
UNSUPPORTED_API     shieldUtils.telemetry is      This API has not been implemented by Firefox.          background.js   45     5
                    not supported
UNSUPPORTED_API     shieldUtils.endStudy is not   This API has not been implemented by Firefox.          background.js   55     7
                    supported
UNSUPPORTED_API     shieldUtils.bootstrapStudy    This API has not been implemented by Firefox.          background.js   69     9
                    is not supported
UNSUPPORTED_API     shieldUtils.info is not       This API has not been implemented by Firefox.          background.js   72     31
                    supported
UNSUPPORTED_API     feature.start is not          This API has not been implemented by Firefox.          background.js   75     9
                    supported
```